server/services/wellcomecollection-api.js

```diff
@@ -5,15 +5,19 @@ import type {Work} from '../model/work';
 const version = 'v1';
 const baseUri = `https://api.wellcomecollection.org/catalogue/${version}`;
 
-export async function getWork(id: string): Promise<Work> {
+export async function getWork(id: string, imageIndex: string): Promise<Work> {
+  const indexParam = imageIndex ? {'_index': imageIndex} : {};
+  const totalQuery = Object.assign(indexParam, {includes: 'identifiers,thumbnail'});
```

Comment:
Strange name give the meaning of `total` - could be (and I did) perceived as the query that returns the total.